### PR TITLE
Update notification handling

### PR DESCRIPTION
**Pull request by bruno-infotrad**
Update code with new notification handlers to prevent notifications from being sent

### CODE REVIEW

**Comment by jdalsem** [comment]
@bruno-infotrad what version of Elgg did this change in?

**Comment by bruno-infotrad** [comment]
From 1.8 to 1.9 (http://learn.elgg.org/en/latest/guides/upgrading.html#notifications)

File: actions/upload/zip.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		$file = $_FILES["zip_file"];
 		
 		// disable notifications of new objects
-		elgg_unregister_event_handler("create", "object", "object_notifications");
+		elgg_unregister_notification_event('object', 'file');
```
Comment (juho-jaakkola) [inline actions/upload/zip.php]: Why does file_tools prevent the file notifications?

Comment (juho-jaakkola) [inline actions/upload/zip.php]: Something to do with the possibility to upload multiple files at the same time?

**Comment by bruno-infotrad** [comment]
Because, if there are 100 files included in the zip file, you will get 100 emails (that is what happened to my client).

**Pull request merged**